eslint.config.mjs

```diff
@@ -118,6 +118,7 @@ export default [
     },
     rules: {
       'yml/no-empty-document': 'off',
+      'yml/no-empty-mapping-value': 'off',
```

Comment:
Why is this needed?

Reply:
linter was complaining when I left a value empty in one of the basic tests I was writing, I will delete it though and make sure it makes sense for future tests